DXRando/DeusEx/Classes/DXRHoverHint.uc

```diff
@@ -0,0 +1,146 @@
+class DXRHoverHint extends Info;
+
+var() string HintText;
+var() int VisibleDistance;
+var Actor target;
+var bool attached;
+
+var Actor baseActor;
+var bool bInWorld;
+
+static function DXRHoverHint Create(Actor a, String hint, vector loc, float rad, float height,optional Name targetName)
```

Comment:
how often do we need to create these by targetName and we don't have an actual reference to that actor instead?